geopandas/geodataframe.py

```diff
@@ -863,6 +875,57 @@ def _to_geo(self, **kwargs):
 
         return geo
 
+    def to_wkb(self, hex=False, **kwargs):
+        """
+        Encode all geometry columns in the GeoDataFrame to WKB.
+
+        Parameters
+        ----------
+        hex : bool
+            If true, export the WKB as a hexadecimal string.
+            The default is to return a binary bytes object.
+        kwargs
+            Additional keyword args will be passed to
+            :func:`pygeos.to_wkb` if pygeos is installed.
```

Comment:
At some point (eg with Shapely 2.0), we should probably just document those keywords